packages/python/plotly/plotly/express/_imshow.py

```diff
@@ -0,0 +1,122 @@
+import plotly.graph_objs as go
+import numpy as np  # is it fine to depend on np here?
```

Comment:
it is fine, but we should have a little friendly error message if it's not installed, similar to the pandas one for px, no?

in fact, px depends on numpy but doesn't have a friendly error message... maybe we can do both in a separate PR.

How about: let's open a separate issue to discuss the `numpy` dependency and move forward with this as is :)